packages/docusaurus/package.json

```diff
@@ -32,23 +32,23 @@
     "@docusaurus/types": "^2.0.0-alpha.32"
   },
   "dependencies": {
-    "@babel/core": "^7.6.4",
+    "@babel/core": "^7.7.2",
     "@babel/plugin-syntax-dynamic-import": "^7.2.0",
-    "@babel/preset-env": "^7.6.3",
-    "@babel/preset-react": "^7.6.3",
+    "@babel/preset-env": "^7.7.1",
+    "@babel/preset-react": "^7.7.0",
     "@docusaurus/utils": "^2.0.0-alpha.32",
     "@endiliey/static-site-generator-webpack-plugin": "^4.0.0",
     "babel-loader": "^8.0.6",
     "babel-plugin-dynamic-import-node": "^2.3.0",
     "cache-loader": "^4.1.0",
     "chalk": "^2.4.2",
-    "chokidar": "^3.2.2",
+    "chokidar": "^3.3.0",
     "classnames": "^2.2.6",
     "commander": "^2.20.0",
-    "copy-webpack-plugin": "^5.0.4",
+    "copy-webpack-plugin": "^5.0.5",
     "core-js": "^2.6.5",
     "css-loader": "^3.2.0",
-    "del": "^4.1.1",
+    "del": "^5.1.0",
```

Comment:
v4 -> v5 most notable breaking changes is Node 6 drop. We're in Node > 8 so should be fine.